community/modules/compute/schedmd-slurm-gcp-v6-partition/variables.tf

```diff
@@ -41,7 +41,12 @@ variable "is_default" {
 }
 
 variable "exclusive" {
-  description = "Exclusive job access to nodes."
+  description = <<-EOD
+    Exclusive job access to nodes. When set to true, only one job can be scheduled on 
+    compute node and nodes are put into power save mode after job finishes. If set to false, 
```

Comment:
When set to true, nodes execute a single job and are deleted after the job exits.